packages/chain-mon/src/balance-mon/service.ts

```diff
@@ -45,7 +47,7 @@ export class BalanceMonService extends BaseServiceV2<
         },
         accounts: {
           validator: validators.str,
-          desc: 'JSON array of [{ address, nickname }] to monitor balances of',
+          desc: 'JSON array of [{ address, nickname, safe }] to monitor balances and nonces of',
```

Comment:
Is this backwards compatible? I think it will be. Its simple to expand the config this way but if we need to add more features to this over time we will likely want a more flexible way to configure in the future

Reply:
it is - accounts without a `safe` field will default to `false`